packages/ui/src/2_molecules/Tabs/components/Tab/Tab.module.css

```diff
@@ -36,13 +36,15 @@
   }
 
   &.secondary {
+    @apply bg-gray-80 mx-1.5;
```

Comment:
I think the first and last child should have 0 margin. we can use gap on parent or remove margin for last-first child here.